### DISCUSSION TITLE
Access Token Returns "Project Not Found"

### DISCUSSION THREAD

**Discussion by rayking99** [iTwin/itwinjs-core#5586] [Q&A]
Hi I have a service account - I can generate an access token and my scope is set to allow read of: 

clashdetection:read changedelements:modify changedelements:read clashdetection:modify insights:modify insights:read realitydataanalysis:modify realitydataanalysis:read users:read projects:modify itwins:modify issues:read validation:modify designelementclassification:read forms:modify designelementclassification:modify projects:read validation:read forms:read issues:modify itwins:read

However, once I generate a token using my key I can't utilise it to get projects or issues. The API returns a status of 201

If I generate the key through the 'Try It Out' function on the website it works no problem - I can call the API etc. I put the API generated token in the same place as the 'Try It Out' token and it returns a 200 OK but it is different - for example if I send the request to https://api.bentley.com/itwins/?subClass=Project with the 'Try It Out' token I get my projects, but with the Api generated token it get: 

```
{
    "iTwins": [],
    "_links": {
        "self": {
            "href": "https://api.bentley.com/itwins/?$skip=0&$top=100&subClass=Project"
        }
    }
}
```

It seems like it doesn't link my App to my Account..

**Comment by pratikshan85** [comment]
@rayking99 Can you please try to add "Client Email" created during App creation to your project and then try.
Please go tto the project details in https://connect.bentley.com and press the manage button . Then add the client email as user and then try.
